components/PerfectGrip/PerfectGrip.module.css

```diff
@@ -20,21 +23,13 @@
   position: relative;
 }
 
-.videoContainer2 {
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  height: 100vh;
-}
-
 .video {
   position: absolute;
-  transform: translateX(-50%);
```

Comment:
Not using transforms on video elements reduces GPU draw slightly (depends on the browser). This still doesn't solve the issue completely as gasp sets transforms on the animated containers.